<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kero's Serial Port</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="m_send_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>手动发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="m_recv_text">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>721</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_send_cb">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>721</width>
      <height>29</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>fd 0b 00 00 07 03 10 aa aa aa aa aa aa aa aa 00 00 00 01 00 e8</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="m_hexshow_check">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>241</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>十六进制显示</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_stopshow_check">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>停止接收数据</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_baudrate_cb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="m_data_cb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="m_stop_cb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="m_jiou_cb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="m_open_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_port_cb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ttyUSB0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyS0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyS1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyS2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyS3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyUSB1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyUSB2</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="m_hexsend_check">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>261</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>十六进制发送</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>奇偶位</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_stopshowsend_check">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>停止显示发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_autosend_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>自动发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_time_tex">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>331</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>周期：</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
